tests/test_scanner/test_scanner.py

```diff
@@ -24,6 +36,12 @@ def test_run_broadcast_checks(scanner):
     assert 0 == len(scanner.issues)  # goatdroid not using these methods
 
 
+def test_run_file_checks(scanner):
+    scanner.issues = []
+    scanner._run_file_checks()
+    assert 0 == len(scanner.issues)
```

Comment:
I'm assuming this just runs the file checks with no files?

Reply:
@nwalsh1995 maybe add a comment to clear up the intention here.